mafContrib/kneMetrics.py

```diff
@@ -4,7 +4,10 @@
 from lsst.sims.utils import uniformSphere
 import lsst.sims.maf.slicers as slicers
 import glob
-from lsst.sims.photUtils import Dust_values
+
+# FIXME replace dust_values with lsst.sims.photUtils
+#from lsst.sims.photUtils import Dust_values
+from dust_values import Dust_values
```

Comment:
This will break for other users who do not have your dust_values file. 
Another option, which I would prefer, would be to paste your snippet of Dust_values code into this file directly, but call the class something else (My_Dust_values would work), and refer to that. 

On the other hand, Datalab has now updated their copy of MAF and so has Dust_values available as an import. So perhaps this could just entirely be fixed now.
(leaving it like this however will cause `import mafContrib` to fail, just like when a module like george is imported but not available)

Reply:
Fixed, all seems to work fine in DataLab.